### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by olgaSavic**
Unit testing for:

- Create new milestone/problem

- Update milestone

- Delete milestone/label

- Close/open problem

- Link/unlink milestone

### CODE REVIEW

File: scm_project/project/tests.py [hunk 1]
```diff
@@ -1,3 +1,126 @@
...
+        self.project = Project.objects.create(title="Project test", git_repository="https://github.com/olgaSavic/UKS",
+                                              author=self.user)
+        self.problem = Problem.objects.create(title="Problem test", description="Problem description test", project=self.project, reported_by=self.user)
+        self.milestone = Milestone.objects.create(title ="Milestone test", description="Milestone description test", project=self.project)
+        self.color = RGBColorField(colors=['#FF0000', '#00FF00', '#0000FF'])
+        self.client.login(username='newUser', password='testing321')
+        self.client.user = self.user     
+        self.date_created = timezone.now()
+        self.due_date = timezone.now()     
+
```
Comment (borkovacd) [inline scm_project/project/tests.py:25]: Initialization of the database is done well!

File: scm_project/project/tests.py [hunk 2]
```diff
@@ -1,3 +1,126 @@
...
+    	milestone = Milestone.objects.create(title="Test milestone unlink", description="Description test milestone",
+  			project=self.project)
+    	problem = Problem.objects.create(title="Problem1", description="Problem1 description test", 
+    		project=self.project, reported_by=self.user, milestone=milestone)
+    	response = self.client.post(reverse('unlink_milestone', 
+    		kwargs = {'problem_id' : problem.id, 'milestone_id' : milestone.id}))
+    	self.assertEqual(response.status_code, 302)
+    	problem.refresh_from_db()
+    	self.assertEqual(problem.milestone, None)
+
```
Comment (borkovacd) [inline scm_project/project/tests.py:120]: All tests are working and done well!

**Review by borkovacd — APPROVED** [review]
Testing branch can be merged with development!

**Comment by olgaSavic** [comment]
Merging testing and development

**Pull request closed without merging**